### DISCUSSION TITLE
Option to require new window

### DISCUSSION THREAD

**Discussion by timkelty** [sebastian-lenz/craft-linkfield#193] [Ideas]
For a field that you want to require to be external, it would be great if you could not only "Allow links to open in new window", but _require_ them to.

Related: https://github.com/sebastian-lenz/craft-linkfield/issues/68

In the settings, perhaps both "Allow links…" and "Allow custom link test" could have nested "required" checkboxes appear beneath them when checked.

On the field front-end, the "open in new window" could be checked and dimmed out (read-only).

**Comment by robmcfadden** [comment]
It would also be nice to still allow a user to make the selection but default to opening in a new window.
